src/material.cpp

```diff
@@ -830,10 +849,25 @@ void Material::calculate_neutron_xs(Particle& p) const
     int i_nuclide = nuclide_[i];
 
     // Calculate microscopic cross section for this nuclide
-    const auto& micro {p.neutron_xs(i_nuclide)};
+#ifndef NCRYSTAL
+    const
+#endif
+    auto& micro {p.neutron_xs(i_nuclide)};
```

Comment:
I would prefer the more-verbose and more-obvious:
```suggestion
#ifdef NCRYSTAL
    auto& micro {p.neutron_xs(i_nuclide)};
#else
    const auto& micro {p.neutron_xs(i_nuclide)};
#endif
```